packages/safe-core-sdk/scripts/generateTypechainFiles.ts

```diff
@@ -0,0 +1,79 @@
+const { execSync } = require('child_process')
```

Comment:
I don't really get why are we optimizing this, this won't end up in the customer's production bundle because these are type definitions. What are we saving? 1 second from type generating or few kilobytes of a developer bandwidth?

Reply:
What it optimizes is not worth it compared to what complexity it brings